changelogs/fragments/76867-service_facts-incomplete-warnings.yaml

```diff
@@ -0,0 +1,2 @@
+bug_fixes:
```

Comment:
Use `bugfixes:` instead and sanity checks will be okay.

Reply:
Thanks..I always screw something up 😂